querybook/webapp/components/DataDocScheduleList/DataDocScheduleList.tsx

```diff
@@ -28,6 +51,16 @@ function useDataDocScheduleFiltersAndPagination() {
         initFilters.scheduled_only ?? false
     );
 
+    const [extraFilters, setExtraFilters] = useState({
```

Comment:
typing is missing
also, make scheduledOnly part of this, no need to have separate setters for that